src/AtelierTomato.Markov.Storage.Sqlite/SqliteSentenceAccess.cs

```diff
@@ -17,7 +17,7 @@ public SqliteSentenceAccess(IOptions<SqliteAccessOptions> options)
 
 		public async Task DeleteSentenceRange(SentenceFilter filter, string? searchString = null)
 		{
-			if ((filter.OIDs is null || !filter.OIDs.Any()) && (filter.Authors is null || !filter.Authors.Any()) && searchString is null)
+			if (filter.OIDs.ToList() is null or [] && filter.Authors.ToList() is null or [] && searchString is null)

```

Comment:
why .ToList() ? i think .ToList() can't give null ... and if OIDs is null then it will cause a nullreferenceexception. weird code.